Homework#2_2.js

```diff
@@ -0,0 +1,19 @@
+function fun1(n) {
+    if (n <= 0) {
+        throw 'Bad  param';
+    }
+    var str1 = 'tick';
+    var str2 = 'tack';
+    for (var i = 1; i <= n; i++) {
+
+        if (i < 10 && i % 3 == 0) {
+            console.log(str1);
+        } else if (i > 10 && i % 5 == 0) {
+            console.log(str2);
+        }
+
+    }
+}
+
+// console.assert(fun1(3)==='tick');
```

Comment:
Make sense to return an object from that function.
This object should contain two properties (tick and tack).
In that case you can easily unit test that.

For example:
```js
function fn1() {
var tickCount = 0;
var tackCount = 0;
// some logic here
return {
tick: tickCount,
tack: tackCount
};
}
```